src/po/tr.po

```diff
@@ -0,0 +1,7284 @@
+msgid "auto-removing autocommand: %s <buffer=%d>"
+msgstr "otokomut kendiliğinden kaldırılıyor: %s <buffer=%d>"
+
+#, c-format
+msgid "E367: No such group: \"%s\""
+msgstr "E367: Böyle bir grup yok: \"%s\""
+
+msgid "E936: Cannot delete the current group"
+msgstr "E936: Mevcut grup silinemiyor"
+
+msgid "W19: Deleting augroup that is still in use"
+msgstr "W19: Kullanımda olan otokomut grubu siliniyor"
+
+#, c-format
+msgid "E215: Illegal character after *: %s"
+msgstr "E215: * sonrası izin verilmeyen karakter: %s"
+
+#, c-format
+msgid "E216: No such event: %s"
+msgstr "E216: Böyle bir olay yok: %s"
+
+#, c-format
+msgid "E216: No such group or event: %s"
+msgstr "E216: Böyle bir grup veya olay yok: %s"
+
+msgid ""
+"\n"
+"--- Autocommands ---"
+msgstr ""
+"\n"
+"--- Otokomutlar ---"
+
+#, c-format
+msgid "E680: <buffer=%d>: invalid buffer number "
+msgstr "E680: <buffer=%d>: geçersiz arabellek numarası"
+
+msgid "E217: Can't execute autocommands for ALL events"
+msgstr "E217: TÜM olaylar için otokomutlar çalıştırılamıyor"
+
+msgid "No matching autocommands"
+msgstr "Eşleşen otokomut yok"
+
+msgid "E218: autocommand nesting too deep"
+msgstr "E218: Çok fazla iç içe geçmiş otokomut"
+
+#, c-format
+msgid "%s Autocommands for \"%s\""
+msgstr "\"%s\" için %s otokomutlar"
+
+#, c-format
+msgid "Executing %s"
+msgstr "%s çalıştırılıyor"
+
+#, c-format
+msgid "autocommand %s"
+msgstr "%s otokomutu"
+
+msgid "E831: bf_key_init() called with empty password"
+msgstr "E831: bf_key_init() boş bir şifre ile çağrıldı"
+
+msgid "E820: sizeof(uint32_t) != 4"
+msgstr "E820: sizeof(uint32_t) != 4"
+
+msgid "E817: Blowfish big/little endian use wrong"
+msgstr "E817: Blowfish yüksek/düşük son haneli kullanımı yanlış"
+
+msgid "E818: sha256 test failed"
+msgstr "E818: sha256 testi başarısız oldu"
+
+msgid "E819: Blowfish test failed"
+msgstr "E819: Blowfish testi başarısız oldu"
+
+msgid "[Location List]"
+msgstr "[Konum listesi]"
+
+msgid "[Quickfix List]"
+msgstr "[Yama listesi]"
+
+msgid "E855: Autocommands caused command to abort"
+msgstr "E855: Otokomutlar komutun durmasına neden oldu"
+
+msgid "E82: Cannot allocate any buffer, exiting..."
+msgstr "E82: Arabellek ayrılamadı, çıkılıyor..."
+
+msgid "E83: Cannot allocate buffer, using other one..."
+msgstr "E83: Arabellek ayrılamadı, başka bir tane kullanılıyor..."
+
+msgid "E931: Buffer cannot be registered"
+msgstr "E931: Arabellek kaydedilemedi"
+
+#, c-format
+msgid "E937: Attempt to delete a buffer that is in use: %s"
+msgstr "E937: Kullanımda olan bir arabellek silinmeye çalışılıyor: %s"
+
+msgid "E515: No buffers were unloaded"
+msgstr "E515: Hiçbir arabellek kaldırılmadı"
+
+msgid "E516: No buffers were deleted"
+msgstr "E516: Hiçbir arabellek silinmedi"
+
+msgid "E517: No buffers were wiped out"
+msgstr "E517: Hiçbir arabellek yok edilmedi"
+
+#, c-format
+msgid "%d buffer unloaded"
+msgid_plural "%d buffers unloaded"
+msgstr[0] "%d arabellek kaldırıldı"
+msgstr[1] "%d arabellek kaldırıldı"
```

Comment:
Does Turkish have plural form?
You are translating singular (msgstr[0]) and plural (msgstr[1]) in the same form.

Reply:
No, both singular and plural forms use the same grammatical structure.

Reply:
> Looks like you are still working on this. Let me know when it's ready to include.

Got it! I will let you know!